docs/changelog.rst

```diff
@@ -1,7 +1,17 @@
 Changelog
 ---------
--  2.0.2 (UNRELEASED) - Drop support to python 3.4
+-  2.0.3 (UNRELEASED)
+-  2.0.2 - Add a `rewind` method to reset a cassette (thanks @khamidou)
```

Comment:
I thought because of the enhanced matcher behaviour this becomes 2.1.0 according to SemVer?

Reply:
You're right ! Just getting the version as it was set previously.